modules/module-extras.php

```diff
@@ -27,10 +27,20 @@
 	'verification-tools/verification-tools-utils.php',
 );
 
+// Not every tool needs to be included if Jetpack is inactive and not in development mode
+if ( ! Jetpack::is_active() && ! Jetpack::is_development_mode() ) {
+	$tools = array(
+		'seo-tools/jetpack-seo-utils.php',
+		'seo-tools/jetpack-seo-titles.php',
+		'seo-tools/jetpack-seo-posts.php',
+	);
+}
+
 /**
  * Filter extra tools (not modules) to include.
  *
  * @since 2.4.0
+ * @since 5.4.0 can be used when Jetpack is not connected to WordPress.com and not in development mode.
```

Comment:
Only in multisite though, otherwise this file will never be loaded as it's blocked [here](https://github.com/Automattic/jetpack/pull/7900/files#diff-9ba9fa5bb52bd6cf36c6c0bdd5ed4830R1657)